dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/UiPluginService.java

```diff
@@ -30,4 +32,6 @@ public interface UiPluginService {
 
     Map<String, Object> queryUiPluginDetailById(int id);
 
+    Map<String, Object>  queryProductInfo(User loginUser, int userId);
```

Comment:
Please use a specific object rather than `Map<String, Object> as the service method result.